inductiva/resources/machine_cluster.py

```diff
@@ -15,7 +15,7 @@ class MPICluster(machines_base.BaseMachineGroup):
     def __init__(self,
                  machine_type: str,
                  num_machines: int = 2,
-                 disk_size_gb: int = 70,
+                 data_disk_gb: int = 0,
```

Comment:
Will this work with `60+0`?
Shouldn't we have a sensible value for data? maybe `10 Gb`?

Reply:
I agree with the sensible size. Will change it